FlowCrypt/Functionality/FilesManager/FilesManager.swift

```diff
@@ -25,6 +25,9 @@ protocol FilesManagerType {
 
     @discardableResult
     func selectFromFilesApp(from viewController: FilesManagerPresenter & UIDocumentPickerDelegate) -> Future<Void, Error>
+
+    func saveLocally(file: FileType) -> URL?
+    func remove(file: FileType)
 }
 
 class FilesManager: FilesManagerType {
```

Comment:
What is the difference between `save`, `saveFilesToApp`, `saveLocally`? Methods should be renamed to make the distinction clearer, seems all the same to me.

Also should be refactored as async methods.